### DISCUSSION TITLE
Microsoft store

### DISCUSSION THREAD

**Discussion by JusMe1** [ciderapp/Cider#295] [Q&A] [answered]
I paid the $1.04 to microsoft store but there was no download!

**Comment by quacksire** [comment, marked as answer]
Not our problem. Please refer to [Microsoft Support](https://duckduckgo.com/?q=microsoft+store+no+download+button&ia=web)

**Comment by JusMe1** [comment]
Roger that. I got it running but it wont open ipod... Good luck with the
project.

**Comment by NEXTOPzlz** [comment]
dowloded from Microsft Store but cannot open on the Win 11
